pyproject.toml

```
@@ -1,12 +1,12 @@
 [project]
-name = "metatensor-models"
+name = "metatrain"
 dynamic = ["version"]
 requires-python = ">=3.9"
 
 readme = "README.md"
 license = {text = "BSD-3-Clause"}
-description = "Atomistic models using metatensor"
-authors = [{name = "metatensor-models developers"}]
+description = "Train machine learning models for atomistic systems."
```

I would go with "training", mainly because it feels better to me ^_^. Feel free to disregard!

```suggestion
description = "Training machine learning models for atomistic systems."
```

---

Thanks!